shared/AppInsightsCore/src/JavaScriptSDK/HelperFuncs.ts

```diff
@@ -0,0 +1,537 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import {  strShimUndefined, strShimObject, strShimPrototype, strShimFunction } from "@microsoft/applicationinsights-shims";
+
+// RESTRICT and AVOID circular dependencies you should not import other contained modules or export the contents of this file directly
+
+// Added to help with minfication
+const strOnPrefix = "on";
+const strAttachEvent = "attachEvent";
+const strAddEventHelper = "addEventListener";
+const strDetachEvent = "detachEvent";
+const strRemoveEventListener = "removeEventListener";
+
+export function objToString(obj: any) {
```

Comment:
Moved from CoreUtils.ts -- to avoid cyclic dependencies -- we have had these build warnings for a while, but this fixes that.